storage/migration.go

```diff
@@ -0,0 +1,129 @@
+/*
+Copyright © 2020 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package storage
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+// MigrationVersion represents a version of the database.
+type MigrationVersion int
```

Comment:
I'd use uint - less conditions in the rest of code

Reply:
Fair enough. I rarely use it because almost all stdlib functions return `int` (and many languages 1. use negative values for errors and 2. don't like implicit conversions between signed and unsigned integers). I'm also not sure how the database driver handles it.

Reply:
Fair enough. Btw IIRC there were proposals to change some stdlib functions to use _proper_ types.

Reply:
`MigrationVersion` is now `uint`, so resolved as far as I can tell